western/index.json

```diff
@@ -2085,5 +2085,501 @@
+      "is_ray_helper": false,
+      "lines": [[[3.369006, 71.100028], [3.382736, 71.28025], [3.392528, 71.418556], [3.424067, 71.684694], [3.444603, 72.205722], [3.523592, 72.169889], [3.526103, 72.017583], [3.524553, 71.859417], [3.505394, 71.863889], [3.424067, 71.684694]]]
+    },
+    {
+      "id": "TA3",
+      "common_name": {"english": "Mini-Cassiopeia (Kemble 2)"},
+      "is_ray_helper": false,
+      "lines": [[[18.579964, 72.535806], [18.571139, 72.411194], [18.598583, 72.380333], [18.58755, 72.274167], [18.616283, 72.267167]]]
+    },
+    {
+      "id": "TA4",
+      "common_name": {"english": "Napoleon's Hat"},
+      "is_ray_helper": false,
+      "lines": [[[14.26071, 18.620694], [14.255533, 18.582861], [14.248164, 18.604861], [14.244167, 18.60625], [14.243525, 18.572972], [14.243797, 18.503778], [14.241222, 18.42975]]]
+    },
+    {
+      "id": "TA5",
+      "common_name": {"english": "The Hercules Keystone"},
+      "is_ray_helper": false,
+      "lines": [[[0.364164, -22.643278], [0.364622, -23.008667], [0.346831, -23.630222], [0.319208, -24.252722]], [[0.346831, -23.630222], [0.372222, -23.840528], [0.373797, -24.266083], [0.412769, -24.286]], [[0.372222, -23.840528], [0.403697, -23.403472], [0.413394, -23.454278]], [[0.403697, -23.403472], [0.364622, -23.008667]]]
+    },
+    {
+      "id": "TA6",
+      "common_name": {"english": "Pouring Cup"},
+      "is_ray_helper": false,
+      "lines": [[[2.216658, 8.846667], [2.205889, 8.788667], [2.198106, 8.701083], [2.189144, 8.569083], [2.180364, 8.368583], [2.150194, 8.4245], [2.138727, 8.50225], [2.132044, 8.629722], [2.127094, 8.846389], [2.138756, 8.850028], [2.174417, 8.864472], [2.200461, 8.899778], [2.216658, 8.846667]], [[2.150194, 8.4245], [2.146967, 8.550972], [2.138756, 8.850028]], [[2.146967, 8.550972], [2.159439, 8.61925], [2.178947, 8.695889], [2.185078, 8.660639], [2.189144, 8.569083]]]
+    },
+    {
+      "id": "TA7",
+      "common_name": {"english": "Mini-Dipper"},
+      "is_ray_helper": false,
+      "lines": [[54061, 53860, 53463, 52877, 52353, 52425, 50685], [52353, 50933, 50685, 49765, 47594, 46977]]
+    },
+    {
+      "id": "TA8",
+      "common_name": {"english": "The Stargate"},
+      "is_ray_helper": false,
+      "lines": [[[12.5925, -12.02111], [12.597981, -12.068889], [12.5973389, -11.9803], [12.5925, -12.02111], [12.59541, -12.024861], [12.597981, -12.068889]], [[12.59541, -12.024861], [12.5973389, -11.9803]]]
+    },
+    {
+      "id": "TA9",
+      "common_name": {"english": "The Jaws"},
+      "is_ray_helper": false,
+      "lines": [[[12.65791, -11.169306], [12.6475861, -11.219306], [12.6462139, -11.238472], [12.6437861, -11.321528], [12.6427, -11.358889], [12.636744, -11.351528], [12.6437861, -11.321528]], [[12.6427, -11.358889], [12.641483, -11.4465], [12.642631, -11.5381], [12.6388583, -11.516861], [12.641483, -11.4465]]]
+    },
+    {
+      "id": "TA10",
+      "common_name": {"english": "Red-Necked Emu"},
```

Comment:
Might need to move to other sky culture (for a later stage work)?

Reply:
What do you mean? I took these all from "Modern" in desktop Stellarium.

Reply:
No, it's in principle OK in "modern" SC, like probably all TAs. The issue with our collection of "western/modern" asterisms (naked-eye and esp. TAs) is that this is a hodge-podge collection of probably everything ever published in any place on earth in popular literature since maybe 1900/1930 (whenever astronomy was widespread enough to allow popular journals come up), or even just online since 1991, and brought to our attention. (We have certainly missed input from amateur communities in many countries!) US American sources popularize slightly other asterisms than Austrian/German, Italian, British or French books.  I assume (but cannot say without a source entry that would add much value to the collection!) that this has been added by an Australian telescope user after 1980 or so. The group is in Cygnus. While it may have gained attention among Australians, I have never read about this in European books or heard it being mentioned in star parties, although we "modern" people have seen Emus on TV or even have them in a nearby zoo. It's just not in everyday's experience.

Reply:
OK. In Desktop Stellarium, switching asterisms "on" shows everything, and many asterisms overlap. Dipper (US)/Plough (UK?)/Wagon (DE/AT) are regionally labeled variants of the same asterism. No Austrian ever calls it "Schöpflöffel", just "Großer Wagen".  Winter-G and Winter Hexagon share 6 stars. etc. Here I would like to see user-selection with permanent storing, per-figure or per-source, just as users like to see what they have learned, and may dislike "suspect" entries like defunct ex-constellations (purged by IAU) too inconspicuous to care about. Others would like to highlight exactly those! This requires some dynamic list UI.

Reply:
Clearly it's not possible to display everything at once, it doesn't make any sense to me, so hiding all is fine for me. Should be also done on desktop I think. Suspect entries are OK if hidden unless searched I guess